src/core/agent/services/ipexCommunicationService.test.ts

```diff
@@ -2127,6 +2101,31 @@ describe("IPEX communication service of agent", () => {
     expect(connections.resolveOobi).toBeCalledTimes(1);
   });
 
+  test("Can create linked ipex message record with history type is credential admitted", async () => {
```

Comment:
Do we already have a test for this?

Reply:
This test for update connection is done when long running operation is completed because that time message is `admitExchange`